spec/uploaders/large_cover_uploader_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require 'carrierwave/test/matchers'
+
+describe LargeCoverUploader do
+  include CarrierWave::Test::Matchers
+
+  let(:video) { Fabricate(:video) }
+
+  before do
+    LargeCoverUploader.enable_processing = true
+    @uploader = LargeCoverUploader.new(video, :cover)
+
+    File.open(File.join(Rails.root, 'public', 'tmp', 'family_guy.jpg')) do |f|
+      @uploader.store!(f)
+    end
+  end
+
+  after do
+    LargeCoverUploader.enable_processing = false
+    @uploader.remove!
+  end
+
+  it "should make the image readable only to the owner and not executable" do
```

Comment:
True, testing for the correct permission settings isn't testing code I wrote. These tests were copied from the examples from the carrierwave documentation. I'll remove these tests.